docs/canonicalk8s/charm/howto/ceph-csi.md

```diff
@@ -129,7 +129,107 @@ sudo k8s kubectl wait pod/pv-writer-test \
     --timeout 2m
 ```
 
+## Relate to multiple Ceph clusters
+
+So far, this guide demonstrates to how to integrate with a single ceph cluster
+represented by the single `ceph-mon` application. But {{product}} supports
+multiple ceph clusters via the same set of charms deployed again as different
+applications.
+
+Deploy an alternate Ceph cluster containing one monitor and one storage unit
+(OSDs). Again, lets limit the resources allocated.
+
+```
+juju deploy -n 1 ceph-mon-alt ceph-mon \
+    --constraints "cores=2 mem=4G root-disk=16G" \
+    --config monitor-count=1 \
+    --config expected-osd-count=1
+juju deploy -n 1 ceph-osd-alt ceph-osd \
+    --constraints "cores=2 mem=4G root-disk=16G" \
+    --storage osd-devices=1G,1 --storage osd-journals=1G,1
+juju deploy ceph-csi-alt ceph-csi \
+    --config provisioner-replicas=1
+juju integrate ceph-csi-alt k8s:ceph-k8s-info
+juju integrate ceph-csi-alt ceph-mon-alt:client
+juju integrate ceph-osd-alt:mon ceph-mon-alt:osd
+```
+
+These applications still uses the same charms, but represent new application
+instances.  A new ceph-cluster via `ceph-mon-alt` and `ceph-osd-alt` and a new
+integration with kubernetes by `ceph-csi-alt`.
+
+There are some Kubernetes Resources which collide in this deployment style
+and the admin will notice the `ceph-csi-alt` charm blocks because those conflict
+are detected.
+
+The `ceph-csi-alt` unit may end up in the `blocked` state with a status:
+
+example)
+`10 Kubernetes resource collisions (action: list-resources)`
+
+By running an action on the charm, we can list those collisions:
```

Comment:
It is suggested to leave we out of how-to guides. 
List the collisions by running an action on the charm: